Network/netx-getting-started-guide.md

```diff
@@ -0,0 +1,273 @@
+{{{
```

Comment:
@jwebs1 just making sure the endpoint locations/kb is 100% accurate still that's all.